<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-45D2DID" failures="2" tests="3" name="Test_Listner" time="0.128" errors="0" timestamp="2022-07-25T16:49:12 IST">
  <testcase classname="retryFailedTestCase.RetryDemolist" name="testCase1" time="0.033">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at retryFailedTestCase.RetryDemolist.testCase1(RetryDemolist.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testCase1 -->
  <testcase classname="retryFailedTestCase.RetryDemolist" name="testCase2" time="0.002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at retryFailedTestCase.RetryDemolist.testCase2(RetryDemolist.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testCase2 -->
  <testcase classname="retryFailedTestCase.RetryDemolist" name="testCase3" time="0.002"/>
</testsuite> <!-- Test_Listner -->
